chanzuckerberg/idseq-web
# Yf/default params for host

Test Plan
- Go to http://localhost:3000/host_genomes add/edit genome
- Create a sample without specifying queue/memory, default value will be set. 
- Bulk uploaded samples should have queue/memory set the proper level based on host genome.
